core/src/test/java/tech/tablesaw/aggregate/AggregateFunctionsTest.java

```diff
@@ -323,7 +323,12 @@ void testMultipleColumnTypes() {
     StringColumn stringColumn = StringColumn.create("s", strings);
 
     Table table = Table.create("test", booleanColumn, numberColumn);
-    table.summarize(booleanColumn, numberColumn, countTrue, stdDev).by(stringColumn);
+    Table summary = table.summarize(booleanColumn, numberColumn, countTrue, stdDev).by(stringColumn);
+    
+    assertEquals(2.0, summary.doubleColumn(1).get(0), 1e-6, "Wrong summary");
+    assertEquals(0.0, summary.doubleColumn(1).get(1), 1e-6,  "Wrong summary");
+    assertEquals(Math.sqrt(2), summary.doubleColumn(2).get(0), 1e-6,  "Wrong summary");
+    assertEquals(Math.sqrt(2), summary.doubleColumn(2).get(1), 1e-6,  "Wrong summary");    
```

Comment:
"Wrong summary" doesn't add anything and my memory is a bit fuzzy, but I think it might also hide what the actual values were
```suggestion
    assertEquals(2.0, summary.doubleColumn(1).get(0), 1e-6);
    assertEquals(0.0, summary.doubleColumn(1).get(1), 1e-6);
    assertEquals(Math.sqrt(2), summary.doubleColumn(2).get(0), 1e-6);
    assertEquals(Math.sqrt(2), summary.doubleColumn(2).get(1), 1e-6);    
```